docs/source/additional/release.rst

```diff
@@ -327,6 +372,264 @@ These sets of versions have been tested for compatibility:
 Changelog
 ---------
 
+Substra 0.22.0 - 2022-10-20
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Main changes
+
+- **BREAKING CHANGE**: the backend type is now set in the Client, the env variable DEBUG_SPAWNER is not used anymore. Default value is deployed (#287)
+
+before:
+
+.. code-block:: python
+
+  export DEBUG_SPAWNER=subprocess
+  client = substra.Client(debug=True)
+
+after:
+
+.. code-block:: python
+
+  client = substra.Client(backend_type=substra.BackendType.LOCAL_SUBPROCESS)
+
+- **BREAKING CHANGE**: schemas.ComputePlanSpec clean_models property is now removed, the transient property on tasks outputs should be used instead.
+- **BREAKING CHANGE**: Model.category field has been removed
+- **BREAKING CHANGE**: train and predict method of all substrafl algos now takes datasamples as argument instead of X abd y. This is impacting the user code only if he or she overwrite those methods instead of using the _local_train and _local_predict methods.
```

Comment:
```suggestion
- **BREAKING CHANGE**: train and predict method of all substrafl algos now takes datasamples as argument instead of X and y. This is impacting the user code only if he or she overwrite those methods instead of using the _local_train and _local_predict methods.
```